### ISSUE TITLE
Update Error: unable to parse manifest

### ISSUE THREAD

**Issue by Felina-Lain** [Norbyte/bg3se#118]
![image](https://github.com/Norbyte/bg3se/assets/5339303/f38ab3ad-126c-4362-ae49-b09f3faed614)

What can I do to solve this issue?
I'm not sure if that's linked but I had a bug earlier where a weird empty input box had popped on the menu screen and wouldn't close or let me send, and I had to restart the game.

The error started after that.

**Comment by gavinlam** [comment]
I'm seeing this in the logs:

`[27-10-2023 19:21:13:136][D:\Jenkins\workspace\Repo\FW4\Release\Stable\LSProjects\Apps\Gustav\Code\EoCServer\Server\GameStateMachine.cpp 67 esv::GameStateMachine::SetTargetState]: esv::GameStateQueued: Uninitialized

[27-10-2023 19:21:13:136][D:\Jenkins\workspace\Repo\FW4\Release\Stable\LSProjects\Apps\Gustav\Code\EoCServer\Server\GameStateMachine.cpp 86 esv::GameStateMachine::Update]: SERVER STATE SWAP - from: Unknown, to: Uninitialized

[27-10-2023 19:21:13:793][D:\Jenkins\workspace\Repo\FW4\Release\Stable\LSProjects\Apps\Gustav\Code\EoCClient\Client\GameStateMachine.cpp 132 ecl::GameStateMachine::SetTargetState]: ecl::GameStateQueued: InitMenu
[27-10-2023 19:21:13:811][D:\Jenkins\workspace\Repo\FW4\Release\Stable\LSProjects\Apps\Gustav\Code\EoCClient\Client\GameStateMachine.cpp 132 ecl::GameStateMachine::SetTargetState]: ecl::GameStateQueued: StartLoading
[27-10-2023 19:21:13:811][D:\Jenkins\workspace\Repo\FW4\Release\Stable\LSProjects\Apps\Gustav\Code\EoCClient\Client\GameStateMachine.cpp 132 ecl::GameStateMachine::SetTargetState]: ecl::GameStateQueued: Init
[27-10-2023 19:21:13:811][D:\Jenkins\workspace\Repo\FW4\Release\Stable\LSProjects\Apps\Gustav\Code\EoCClient\Client\GameStateMachine.cpp 132 ecl::GameStateMachine::SetTargetState]: ecl::GameStateQueued: LoadModule
[27-10-2023 19:21:13:811][D:\Jenkins\workspace\Repo\FW4\Release\Stable\LSProjects\Apps\Gustav\Code\EoCClient\Client\GameStateMachine.cpp 132 ecl::GameStateMachine::SetTargetState]: ecl::GameStateQueued: LoadMenu
[27-10-2023 19:21:13:811][D:\Jenkins\workspace\Repo\FW4\Release\Stable\LSProjects\Apps\Gustav\Code\EoCClient\Client\GameStateMachine.cpp 132 ecl::GameStateMachine::SetTargetState]: ecl::GameStateQueued: StopLoading
[27-10-2023 19:21:13:811][D:\Jenkins\workspace\Repo\FW4\Release\Stable\LSProjects\Apps\Gustav\Code\EoCClient\Client\GameStateMachine.cpp 115 ecl::GameStateMachine::SetTargetStateAfterFade]: ecl::GameStateQueued: Menu after fade: FadeInFadeOut
[27-10-2023 19:21:14:158][D:\Jenkins\workspace\Repo\FW4\Release\Stable\LSProjects\Apps\Gustav\Code\EoCClient\Client\GameStateMachine.cpp 250 ecl::GameStateMachine::Update]: CLIENT STATE SWAP - from: Unknown, to: InitMenu
[27-10-2023 19:21:14:974][D:\Jenkins\workspace\Repo\FW4\Release\Stable\LSProjects\Apps\Gustav\Code\EoCClient\Client\GameStateMachine.cpp 250 ecl::GameStateMachine::Update]: CLIENT STATE SWAP - from: InitMenu, to: StartLoading
[27-10-2023 19:21:16:436][D:\Jenkins\workspace\Repo\FW4\Release\Stable\LSProjects\Apps\Gustav\Code\EoCClient\Client\GameStateMachine.cpp 250 ecl::GameStateMachine::Update]: CLIENT STATE SWAP - from: StartLoading, to: Init
[27-10-2023 19:21:18:020][D:\Jenkins\workspace\Repo\FW4\Release\Stable\LSProjects\Apps\Gustav\Code\EoCClient\Client\GameStateMachine.cpp 250 ecl::GameStateMachine::Update]: CLIENT STATE SWAP - from: Init, to: LoadModule
[27-10-2023 19:21:21:728][D:\Jenkins\workspace\Repo\FW4\Release\Stable\LSProjects\Framework\Code\GameNet\PortMapperUPnP.cpp 115 net::PortMapperUPnP::Discover]: [NET] Discovery finished... no usable UPnP enabled routers found
[27-10-2023 19:21:34:188][D:\Jenkins\workspace\Repo\FW4\Release\Stable\LSProjects\Apps\Gustav\Code\EoCClient\Client\GameStateMachine.cpp 250 ecl::GameStateMachine::Update]: CLIENT STATE SWAP - from: LoadModule, to: LoadMenu
[27-10-2023 19:21:43:541][D:\Jenkins\workspace\Repo\FW4\Release\Stable\LSProjects\Apps\Gustav\Code\EoCClient\Client\GameStateMachine.cpp 250 ecl::GameStateMachine::Update]: CLIENT STATE SWAP - from: LoadMenu, to: StopLoading
[27-10-2023 19:21:44:075][D:\Jenkins\workspace\Repo\FW4\Release\Stable\LSProjects\Apps\Gustav\Code\EoCClient\Client\GameStateMachine.cpp 250 ecl::GameStateMachine::Update]: CLIENT STATE SWAP - from: StopLoading, to: Menu
[27-10-2023 19:21:44:116][D:\Jenkins\workspace\Repo\FW4\Release\Stable\LSProjects\Apps\Gustav\Code\EoCClient\Client\EocClient.cpp 1978 ecl::EocClient::HandleError]: Handling error: Script Extender update failed.

Unable to parse manifest: * Line 1, Column 1
  Syntax error: value, object or array expected.
`

**Comment by gavinlam** [comment]
As a workaround, you can try the manual install: https://bg3se-updates.norbyte.dev/Channels/Release/ManualInstall.zip

**Comment by tbabic** [comment]
How critical is this? Game seems playable despite this happening...

**Comment by cemgorey** [comment]
Same here, doesnt seem to affect the installed mods I have but would love a solution, thanks.

**Comment by karzablade** [comment]
I started getting this error too, been trying to fix it for about an hour. Tried using the manual install and everything.

**Comment by Felina-Lain** [comment]
> As a workaround, you can try the manual install: https://bg3se-updates.norbyte.dev/Channels/Release/ManualInstall.zip

There is a second file in there, and I see no instructions on how to manually install? Where do I put the second file?

**Comment by Riggleton14** [comment]
I got the same error message when I started my game today. Played the game last night without any errors and as far as I've been able to tell there wasn't any updates this morning for BG3.

**Comment by Felina-Lain** [comment]
> I got the same error message when I started my game today. Played the game last night without any errors and as far as I've been able to tell there wasn't any updates this morning for BG3.

I think the update was in the script extender, but it seems to have messed something up?

**Comment by karzablade** [comment]
> > I got the same error message when I started my game today. Played the game last night without any errors and as far as I've been able to tell there wasn't any updates this morning for BG3.
> 
> I think the update was in the script extender, but it seems to have messed something up?

Yes, from what I can tell the script extender updates automatically. So this is likely the issue as we've all experienced the error within a similar time frame- and I can't see any previous error reports about this, so it must be a new thing. Personally I won't be playing until there's a fix! I don't want to break anything within the actual game files.

**Comment by maseren** [comment]
you can add me to the list of people having issues with this. Again, no changes. Just a restart of the PC and the issue occurred.

**Comment by rcx1** [comment]
I can confirm the same error.
But there is no error shown in console / log. so it seems like update / updater issue.

**Comment by DrearburhCrow** [comment]
Same here - seems to be a widespread issue with the updater.

**Comment by VolzanIT** [comment]
same here, and this appned we i decided to installa my first mod so sad

**Comment by gavinlam** [comment]
> > As a workaround, you can try the manual install: https://bg3se-updates.norbyte.dev/Channels/Release/ManualInstall.zip
> 
> There is a second file in there, and I see no instructions on how to manually install? Where do I put the second file?


I put both in the bin directory. I played for an hour without issue using the manual install.

**Comment by Cenoris** [comment]
@gavinlam I signed up to GitHub just to say Thank You. This worked for me. I am most grateful.

**Comment by Riggleton14** [comment]
Manually installing and overriding the old files in the bin folder worked for me.

**Comment by megdorkable** [comment]
I am experiencing this issue as well.

**Comment by tokman2** [comment]
Same error

**Comment by Cyberk999** [comment]
I legit just cant play modded anymore, even with that manual install file. none of my load in. its insane how dumb and complicated this modding scene is

**Comment by karzablade** [comment]
> > > As a workaround, you can try the manual install: https://bg3se-updates.norbyte.dev/Channels/Release/ManualInstall.zip
> > 
> > 
> > There is a second file in there, and I see no instructions on how to manually install? Where do I put the second file?
> 
> I put both in the bin directory. I played for an hour without issue using the manual install.

Thank you! That works somewhat :D I don't get the error anymore but my game keep crashing after like half an hour of playing rip

**Comment by prphawk** [comment]
same error here, and I can't actually play the game! my saves won't load.

**Comment by EduardoFerreira98** [comment]
Same here

**Comment by maseren** [comment]
The new update with the additional file didn't work for existing saves. I
had to go back to the others that give the error but allow you to play on.
Unknown if it's stable.

**Comment by EduardoFerreira98** [comment]
> > As a workaround, you can try the manual install: https://bg3se-updates.norbyte.dev/Channels/Release/ManualInstall.zip
> 
> There is a second file in there, and I see no instructions on how to manually install? Where do I put the second file?

I'm prety sure the second file is only usable for linux comuputers, if you use linus you shoud just drop the DWrite file in the bin folder

**Comment by EduardoFerreira98** [comment]
I just tried the manual  install and now my game crash the moment I try to launch it

**Comment by EduardoFerreira98** [comment]
> I just tried the manual install and now my game crash the moment I try to launch it

Dowloading back the regular extender got the game to launch again. The error message still there but I can still load the save files. But i'm scared to lose progress so I'm not playing ultil this is fixed.

Also don't forget to backup your saves everybody!

**Comment by yzsshen** [comment]
@EduardoFerreira98 place both files in that manual install link into /bin

**Comment by maseren** [comment]
> > I just tried the manual install and now my game crash the moment I try to launch it
> 
> Dowloading back the regular extender got the game to launch again. The error message still there but I can still load the save files. But i'm scared to lose progress so I'm not playing ultil this is fixed.
> 
> Also don't forget to backup your saves everybody!

I did this as well. Went with the old legit one and so far so good. Played an hour and a half with my wife, no issues so far.

**Comment by kimkimH2** [comment]
Same issue here, was working fine with no changes earlier. My saves seem to be working fine but am unsure why this message is there

**Comment by coredumperror** [comment]
Can confirm that performing the manual install as suggested in this thread (download the link suggested by @gavinlam and extract the zip file to the /bin folder in BG3's directory) worked like a charm.

**Comment by Spartacii** [comment]
I disagree that the manual install works. 
All it does is to prevent the error popup from showing up but it is still recorded as:

Unable to parse manifest: * Line 1, Column 1
  Syntax error: value, object or array expected.

In the network log. 
Plus, there's a lot of background stuff in the network log that is no longer showing up with the manual install that used to show up with the original/latest BG3SE.
Preventing the popup from showing up but still being recorded in the network log is just a placebo and not really a fix.

**Comment by XashiMM** [comment]
Signed up to also confirm that this also happened to me just now. Hoping it gets fixed soon.

**Comment by Cuchullain-alldaylong** [comment]
Unfortunately, this bug is affecting the game.  I CTD whenever I attempt to fast travel out of zone.  I just killed an end game character in an older zone, ported to camp, and when I attempted to port back to an Act 3 zone, I crashed to desktop.  I tried uninstalling Script Extender and the 2 mods I have that need it (Camp Event Notification and Bag of Campers' Inventories) and then the game crashed to desktop when attempted to load a saved game without the mods.

I manually installed Script Extender per above instructions.  Reinstalled the two mods and launched the game.  The Script Extender error no longer appeared and I did a very premature celebration.  As soon as I attempted to port to the Act 3 zone, I CTD again.  Everything was working fine on 10/27, no errors of any kind, no crashes, no problems porting to and from different zones.

**Comment by BelegCufea** [comment]
OK guys, it seems the "configuration" file containing version info that is beeing downloaded during Script Extender initialization is corrupted or not available.

## TLDR
Nothing in SE was changed. It shoud work the same as yesterday. The error you see is only "cosmetic". Any strange game behaviour you see is the same that you would encounter yesterday when the error was not there.
Be patient. Norbyte will fix the error soon(TM). You can't do anything to fix the problem.

## 
### Script Extender version
Be aware that SE was **NOT** updated (as of writing this post) from v7 released on October 8th.

### Manual install
The current [https://bg3se-updates.norbyte.dev/Channels/Release/ManualInstall.zip](https://bg3se-updates.norbyte.dev/Channels/Release/ManualInstall.zip) manual file is the same version (October 8th).
This install won't connect to internet to check for updates so you will not see the error, but that's all.
**_Warning_**
If you use the manual installation you will loose the auto update feature of SE and will have to from then on check for new versions (on [https://bg3se-updates.norbyte.dev/Channels/Release/Manifest.json](https://bg3se-updates.norbyte.dev/Channels/Release/Manifest.json) and manualy install them. _You can always revert back to online updates installing `DWrite.DLL` from [https://github.com/Norbyte/bg3se/releases](https://github.com/Norbyte/bg3se/releases)._

**Comment by gavinlam** [comment]
The latest version of Script Extender seems to have resolved this issue. I remove all the files from the manual install and re-installed the normal script extender. The manifest error did not come up.

**Comment by Norbyte** [comment]
I'm closing old issues related to old game versions, since those were largely fixed by later game patches and SE updates and are no longer relevant/reproducible.
